analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -331,6 +329,7 @@ private void align(Configuration configuration, ManipulationCache cache, Manipul
         writeGmeConfigMarkerFile(rootProject.getBuildFile());
         writeGmeReposMarkerFile();
         writeRepositorySettingsFile(cache.getRepositories());
+        final Set<ProjectVersionRef> nonAligned = new LinkedHashSet<>();
```

Comment:
You're wanting to maintain order?

Reply:
Since it maintains order for the other printed dependencies (as caught by a test), then it makes sense not to lose order for the non-aligned as well. Because we print these and/or verify their order, they should be predictable and `HashSet` does not guarantee order.

Reply:
Ok, thanks.